learn/guides/developing-a-restful-api-with-ballerina.md

```diff
@@ -0,0 +1,249 @@
+---
+layout: developing-a-restful-api-with-ballerina
+title: Developing a RESTful API with Ballerina
+permalink: /learn/developing-a-restful-api-with-ballerina/
+description: This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+active: developing-a-restful-api-with-ballerina
+intro: This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
+redirect_from:
+  - /learn/developing-a-restful-api-with-ballerina
+---
+
+This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
```

Comment:
```suggestion
This simple guide helps you understand the basics of Ballerina constructs which allow you to write RESTful APIs. 
```
Please see the review comments in the GrpahQL guide PR: https://github.com/ballerina-platform/ballerina-dev-website/pull/3369